pkg/util/kubernetes/apiserver/controllers/hpa_controller_test.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build !race && kubeapiserver
+//go:build kubeapiserver
```

Comment:
I think this was stopping it from running on CI. I don’t know why it was added. Maybe it was a workaround because the test was flaky on CI.

We can try merging and see if it flakes.